docs/data-cookbooks/throttling-queued-executions.md

```diff
@@ -16,29 +16,70 @@ Limiting the number of executions that can be running from a given queue is usef
 
 ### Create and deploy the queue
 
-#### Define a queue with a maximum number of executions
-
-In your `app/config.yml`, define a new queue with a `maxExecutions` value:
-
-```yaml
-  sqs:
-    backgroundJobQueue:
-      visibilityTimeout: 60
-      retry: 30
-      maxExecutions: 5
-      consumer:
-        - lambda: sqs2sfThrottle    # you must use this lambda
-          schedule: rate(1 minute)
-          messageLimit: '{{sqs_consumer_rate}'
-          state: ENABLED
+#### Add a new queue
+
+In a `.tf` file for your [Cumulus deployment](./../deployment/README.md#deploy-the-cumulus-instance), add a new SQS queue:
```

Comment:
This link doesn't work

Reply:
This one should be this `./../deployment/deployment-readme#deploy-the-cumulus-instance`